docs/miscellaneous/contribute/translate/rewards.md

```diff
@@ -5,7 +5,9 @@ sidebar_position: 6
 
 As any honest and well-performed effort should be rewarded, active translators on Crowdin can request a free AdGuard license key and a VPN subscription.
 
-To request a **one-year AdGuard license key** or a **one-year AdGuard VPN subscription**, you must translate at least **500** and **2,500** words respectively.
+To request a **one-year AdGuard license key** or a **one-year AdGuard VPN subscription**, you must translate at least **500** and **2,500** words respectively. 
+
+To check your translation progress, open a project you have worked on, click the *Reports* tab and select *My Contribution*. You can summarise the number of words translated in different projects to get a license.
```

Comment:
summarise —>  summarize (it's American English)
but I guess 'you can sum up the number of words' is better